### ISSUE TITLE
Sonographic non‐visualized fetal bladder

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7376]
**Preferred term label:**
Sonographic non‐visualized fetal bladder

**Synonyms**

**Definition (free text, please give PubMed ID)**
Inability to visualize the fetal bladder on targeted prenatal sonography examination. The fetal bladder can be visualized with sonography as a midline, fluid-filled structure in the pelvis as early as 10-12 weeks of gestation. The normal fetus voids at least once an hour but never completely empties the urinary bladder. The fetal bladdder should be consistently imaged after 13 weeks of gestation.

PMID:8915088

**Parent term (use hpo.jax.org/app)**
Abnormal fetal morphology HP:0034058 -- should we have more fetal kidney terms here?

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP:0034217
